datatypes/src/operations/image/to_png.rs

```diff
@@ -0,0 +1,177 @@
+use crate::operations::image::{Colorizer, RgbaTransmutable};
+use crate::raster::{GridPixelAccess, Raster, Raster2D};
+use image::{DynamicImage, ImageFormat, RgbaImage};
+
+pub trait ToPng {
+    /// Outputs png bytes of an image of size width x height
+    fn to_png(&self, width: u32, height: u32, colorizer: &Colorizer) -> Vec<u8>;
```

Comment:
i can imagine that this function can fail. So i guess its a trait TryToPng with try_to_png -> Result<...> ?

Reply:
I am not sure if we should prefix try_... to a lot of types since lots of error can happen and this could bloat the code.

There are cases where it makes sense, e.g., Into<T> and TryInto<T> where there is a non-failing type.

4dfc595